streaming/vision/coco.py

```diff
@@ -53,8 +52,8 @@ def __getitem__(self, idx: int) -> Any:
         wtot = x['wtot']
         bbox_sizes = x['bbox_sizes']
         bbox_labels = x['bbox_labels']
+        coco_args = (img, (htot, wtot), bbox_sizes, bbox_labels)
         if self.transform:
-            img, (htot,
-                  wtot), bbox_sizes, bbox_labels = self.transform(img, (htot, wtot), bbox_sizes,
-                                                                  bbox_labels)
+            coco_args = self.transform(*coco_args)
```

Comment:
oof, i didn't notice we are passing img_id as well in the return statement, i think this way looks worse, let's revert it?

Reply:
Yes you are right.

Reply:
Done in https://github.com/mosaicml/streaming/pull/13/commits/63203edee1a388f9ddc7a8edc5c72f700bf297fa.